Simplenote/NoteEditorViewController.m

```diff
@@ -293,8 +293,12 @@ - (void)displayNotes:(NSArray *)notes
 - (void)checkTextInDocument
 {
     dispatch_async(dispatch_get_main_queue(), ^() {
+        // Temporarily remove the editor delegate because `checkTextInDocument`
+        // fires `textDidChange` which will erroneously modify the note
```

Comment:
```objective-c
// Temporarily remove the editor delegate because `checkTextInDocument`
// fires `textDidChange` which will erroneously modify the note's modification
// time and unintentionally change the sort order of the note in the list as a result
```

^^^ how about something like that?

Reply:
Hahah ok, we'll see what @jleandroperez thinks about long comments 😛

Reply:
LOVE long comments, specially if it's hard to understand what the code below does, end to end!!